### ISSUE TITLE
Share Sound

### ISSUE THREAD

**Issue by FestplattenSchnitzel** [pavlobu/deskreen#28]
Share screen **and** sound with the connected device.

Intended use case(s):
- Stream a video from a PC to a (toaster) "Smart" TV, smartphone, tablet, …

**Comment by pavlobu** [comment]
Yes sounds cool but it requires even more OS hacking. Hope more people with specific knowledge will pay attention to this project to implement that feature.
